site/content/concepts/device-lifecycle-notifications.md

```diff
@@ -0,0 +1,93 @@
++++
+title = "Device lifecycle notifications (from 0.6 on)"
+weight = 415
++++
+
+This page describes how device lifecycle notifications are handled in the Eclipse Hono project.
+
+
+{{% warning %}}
+This concept is currently part of active development and discussion.
+The content here summarizes the current discussion and is being prototyped.
+Please join our mailing list or consult the IRC channel in case you have any questions or want to participate.
+{{% /warning %}}
+
+<!--more-->
+
+Business applications connected to Hono may want to invoke arbitrary functionality when a device connects or disconnects to Hono.
+This is especially necessary to implement Command and Control, but is not limited to this Use Case. Integrating digital twins, like
+[Eclipse Ditto] (https://projects.eclipse.org/proposals/eclipse-ditto), also benefit from knowing when a device just connected to Hono.
+
+For these scenarios Hono defines a lifecycle notification concept that covers different patterns that try to cover several classes
```

Comment:
We need to distinguish between connected/disconnected events published by protocol adapters and how a device signals that it can receive a command. These are two different things and should be explained here. FMPOV they are not both part of what is called *lifecycle notifications* here. Additionally, IMHO that term does not match the purpose of either of the two concepts and should be replaced by two other terms.

Reply:
No, as agreed above already. Will replace it by different wording.